app/lib/search/text_utils.dart

```diff
@@ -58,12 +58,7 @@ String compactReadme(String? text) {
 
 String normalizeBeforeIndexing(String? text) {
   if (text == null) return '';
-  final String t = text
-      .toLowerCase()
-      .replaceAll(_nonCharacterRegExp, ' ')
-      .replaceAll(_multiWhitespaceRegExp, ' ')
-      .trim();
-  return t;
+  return text.toLowerCase().replaceAll(_nonCharactersRegExp, ' ').trim();
```

Comment:
Don't you still need the multiWhitespace replacement?

Reply:
As whitespace is also outside of `[a-z0-0]`, we already have it in the first replacement.

Reply:
Ah - I think I get it now